MapboxNavigation/Geometry.swift

```diff
@@ -1,41 +1,40 @@
 import CoreLocation
 import MapboxDirections
 
-typealias LocationRadians = Double
-typealias RadianDistance = Double
-typealias RadianDirection = Double
+public typealias LocationRadians = Double
+public typealias RadianDistance = Double
+public typealias RadianDirection = Double
+let metersPerRadian = 6_373_000.0
 
-extension CLLocationDegrees {
-    func toRadians() -> LocationRadians {
+public extension CLLocationDegrees {
+    public func mb_toRadians() -> LocationRadians {
```

Comment:
Note that it isn’t possible to extend a primitive type like `CLLocationDegrees` in Objective-C, so this extension will be unavailable to Objective-C code.